client/src/components/modals/nr-not-required.vue

```diff
@@ -1,29 +1,35 @@
 <template>
   <!-- NB: dummy modal attach point so unit tests can see it -->
-  <v-dialog v-model="showModal" max-width="40%" attach="">
-    <v-card class="pa-9">
-      <v-card-text class="h3">You don't need a Name Request if...</v-card-text>
+  <v-dialog v-model="showModal" width="50rem" attach="">
+    <v-card class="pa-9 nr-not-required-dialog">
+      <v-card-title>
+        <span>
+          <slot name="title">You don't need a Name Request if...</slot>
+        </span>
+      </v-card-title>
       <v-card-text class="copy-normal">
         <ol>
           <li>
             <b>You are already Federally Incorporated.</b><br>
-            You do not require a name request. Please proceed to COLIN to extra-provincially register your business.
+            You do not require a name request. Please go to <a :href="corpOnlineLink" target="_blank">
```

Comment:
For all these links, was there a requirement to show a "mdi-launch" icon (to indicate this will open in a new page)?
If so, see names-capture.vue.

Reply:
No icon for these links